includes/class-airtable-api.php

```diff
@@ -2,97 +2,140 @@
 
 namespace UbcVpfoSpacesPage;
 
-defined( 'ABSPATH' ) || exit;
+defined('ABSPATH') || exit;
 
```

Comment:
@adampatterson Just a note for future reference that our coding standards scan requires there to be spaces between parentheses and the content of the parentheses in functions.